core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/upscale/ClusterUpscaleContext.java

```diff
@@ -16,14 +16,20 @@
 
     private final ClusterManagerType clusterManagerType;
 
+    private final Boolean repair;
```

Comment:
Why the wrapper type instead of the primitive?

Reply:
because singlePrimaryGateway was Boolean also :D